_overviews/scala3-macros/tutorial/quotes.md

```diff
@@ -249,7 +249,106 @@ case ...
 
 ### Matching types
 
-*Coming soon*
+So far we assumed that we knew statically the type within the quote patterns.
+Quote patterns also allow for generic types and existential types which we will see in this section.
```

Comment:
```suggestion
Quote patterns also allow for generic types and existential types, which we will see in this section.
```

Reply:
Maybe say: "In general, generic types and existential types are not statically known." to connect the two sentences.